packages/@ember/-internals/glimmer/lib/helper.ts

```diff
@@ -197,7 +197,7 @@ class ClassicHelperManager implements HelperManager<ClassicHelperStateBucket> {
   }
 }
 
-setHelperManager((owner) => new ClassicHelperManager(owner), Helper);
+setHelperManager((owner: Owner | undefined) => new ClassicHelperManager(owner!), Helper);
```

Comment:
Seems **very** suspect to update the signature here to allow undefined **and** to use `owner!`.